FeLib/Source/feio.cpp

```diff
@@ -163,15 +163,28 @@ int iosystem::Menu(cbitmap* BackGround, v2 Pos,
   int c = 0;
 
   if(BackGround){
-    //vanilla was 800x600 as the background menu image. TODO provide calculations for lower than 800x600 one day?
-    if(RES.X < BackGround->GetSize().X)ABORT("invalid window width %d",RES.X);
-    if(RES.Y < BackGround->GetSize().Y)ABORT("invalid window height %d",RES.Y);
-
     if( (RES.X!=BackGround->GetSize().X) || (RES.Y!=BackGround->GetSize().Y) ){
+      blitdata B = DEFAULT_BLITDATA;
+      B.Bitmap = &Buffer;
+
+      B.Src.X = (RES.X - BackGround->GetSize().X)/2;
+      if(B.Src.X>0)B.Src.X=0;
+      if(B.Src.X<0)B.Src.X*=-1;
+      B.Src.Y = (RES.Y - BackGround->GetSize().Y)/2;
+      if(B.Src.Y>0)B.Src.Y=0;
+      if(B.Src.Y<0)B.Src.Y*=-1;
```

Comment:
What do Src.X and Src.Y mean?

Reply:
blitdata should have better named var members:
```
  bitmap* Bitmap; //target bitmap to draw in
  v2 Src; //pos at source bitmap to copy from
  v2 Dest; //pos at target bitmap to draw in
  v2 Border; //size to copy from at src bitmap
```
we could just add these comments there too :)